Added missing modelling rules in PackML Nodeset

JGrothoff opened the pull request:
Instance declarations of CurrentState (with Id as child) and UnitModeCurrent didn't keep their modelling rule mandatory.

According to [Spec Part3 6.4.4.4](https://reference.opcfoundation.org/v104/Core/docs/Part3/#6.4.4.4) "all instances that are referenced directly or indirectly from ‘A’ based on InstanceDeclarations of ‘A_Type’ initially maintain the same ModellingRule as their InstanceDeclarations."

JGrothoff committed:
Added missing modelling rules in PackML Nodeset

Instance declarations of CurrentState (with Id as child) and UnitModeCurrent didn't keep their modelling rule mandatory.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/OPCFoundation/UA-Nodeset?pullRequest=66) <br/>All committers have signed the CLA.

JGrothoff committed:
Corrected Cleared and Run(ning) state numbers

According to [Annex B](https://reference.opcfoundation.org/v104/PackML/v100/docs/B.1.1/) the StateNumber values for the macro states Cleared and Running need to be 18 and 19.
'Running' is called 'Run' in the standard, but i don't know whether this is an intended changed.

opcfoundation-org commented:
These kinds of changes need to be submitted as Mantis issues against the specification:
https://apps.opcfoundation.org/mantis/set_project.php?project_id=98&amp;make_default=no

JGrothoff commented:
Ok, i found this issues in existing Mantis issues and linked this PR there. So i will close this PR for now.
